File: 프로그래머스/lv1/42748. K번째수/K번째수.js
function solution(array, commands) {
    // i부터 j까지 자르고, 
    // 정렬
    // k번째에 있는 수를 도출
    var answer = [];
    for(let a = 0; a < commands.length; a++){
        let i = commands[a][0] 
        let j = commands[a][1]
        let k = commands[a][2]
        let new_arr = array.slice(i-1,j)
        new_arr.sort((a,b)=>a-b)
        let res = new_arr[k-1]
        answer.push(res)
    }
    return answer;
}

File: 프로그래머스/1/42748. K번째수/K번째수.py
def solution(array, commands):
    answer = []
    for c in commands:
        i = c[0] - 1
        j = c[1]
        k = c[2] - 1
        A = array[i:j]
        A = sorted(A)
        answer.append(A[k])
    return answer